vita3k/gui/src/user_management.cpp

```diff
@@ -326,7 +326,7 @@ void draw_user_management(GuiState &gui, EmuEnvState &emuenv) {
             }
             user_id_selected = fmt::format("{:0>2d}", id);
             auto i = 1;
-            const auto user = lang["user"].c_str();
+            const auto user = "User";
```

Comment:
revert

Reply:
> revert

actually i revert my change, same like ps4

Reply:
? In french is writer no user, but "Utilisateur1"

Reply:
> ? In french is writer no user, but "Utilisateur1"

ahhh, yes, u r right Mr. Zangetsu, i check again and exist on ps4, only jpn/kor/chn is "User1"